README.md

```diff
@@ -39,10 +39,10 @@ The widget is a ready-to-use drop-in bundle with a CSS and customizable template
 PingFederate acts as the server interacting with the widget via APIs to authenticate the user.
 
 To configure PingFederate for the widget:
-  1. Enable the authentication API: Identity Provider > Authentication Applications > Enable Authentication API.
-  1. Add an application: Add Authentication Application > Name: TestApp, URL:  `https://localhost:8443` > Save > Save.
+  1. Enable the authentication API: Authentication > Authentication API Applications > Enable Authentication API.
+  1. Add an application: Add Authentication Application > Name: TestApp, URL:  `https://localhost:8443` > Save.
   1. Create a Password Credential Validator, HTML Form Adapter, and an IdP Connection that uses the adapter for SSO. Alternatively, use OAuth Playground flows to authenticate using the HTML Form Adapter.
-  1. Create a policy: Policies > Create (or update an existing policy) that uses the HTML Form Adapter > Authentication Application > Select the previously created app from the dropdown.
+  1. Create a policy: Authentication > Policies > Create (or update an existing policy) that uses the HTML Form Adapter > Authentication Application > Select the previously created app from the dropdown.
```

Comment:
@jameslamothe-ping could you have a second look at this line and offer any suggestions? I find it a bit confusing to follow.

Reply:
After pondering the text and playing with PF for 15 minutes, I just don't understand what it's trying to say. Sorry.

Reply:
Ok, let me see if I can simplify these steps and submit something for review.